go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/go-openapi/validate v0.19.11
 	github.com/go-swagger/go-swagger v0.25.0 // indirect
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/ldsec/medco-connector v1.0.0
```

Comment:
What is this added dependency? It's the deprecated repo, it needs to be removed.